csunplugged/plugging_it_in/views.py

```diff
@@ -52,23 +55,61 @@ def get_context_data(self, **kwargs):
 
 
 class TopicView(generic.DetailView):
```

Comment:
I've changed this view to `LessonsView` since it lists all lessons for a topic and also renamed the old `LessonView` to `ProgrammingChallengeListView`. The comments have also been updated accordingly.